addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -516,7 +516,9 @@ var SlideUploadDialog = Dialog.extend({
         if (!preventOnchange) {
             var input = file.name;
             var inputVal = input.substr(0, input.lastIndexOf('.')) || input;
-            this._formSetFieldValue('name', inputVal);
+            if(this._formGetFieldValue('name') == ""){
```

Comment:
You need to test for triple equality in JS.

Is your linter correctly configured?